### PR TITLE
Adding compatibility with tensorflow2

### PR DESCRIPTION
**Pull request by pzinemanas**
This PR is related to #11 and #7. With this implementation, we maintain compatibility with tensorflow 1 and 2.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -48,4 +48,28 @@ jobs:
...
-        verbose: true
\ No newline at end of file
+        verbose: true
+
+  build_tf2:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```
Comment (mrocamora) [inline .github/workflows/main.yml:58]: Shouldn't we add python 3.8? 
https://www.tensorflow.org/install/pip

File: README.md [hunk 2]
```diff
@@ -31,16 +33,13 @@ DCASE-models uses [SoX](http://sox.sourceforge.net/) for functions related to th
...
 conda install -c conda-forge sox
 ```
-Before installing the library, you must install only one of the Tensorflow variants: CPU-only or GPU.
+When installing the library, you must select the tensorflow variant: version 1 (CPU-only or GPU) or version 2.
 ``` 
-pip install "tensorflow<1.14" # for CPU-only version
-pip install "tensorflow-gpu<1.14" # for GPU version
+pip install DCASE-models[keras_tf] # for tensorflow 1 CPU-only version
+pip install DCASE-models[keras_tf_gpu] # for tensorflow 1 GPU version
+pip install DCASE-models[tf2] # for tensorflow 2
```
Comment (mrocamora) [inline README.md:40]: Cool :)

File: setup.py [hunk 3]
```diff
@@ -19,22 +19,22 @@
...
-        'openl3==0.3.1',
-        'scikit-learn==0.22.1',
-        'keras==2.2.4',
-       # 'tensorflow-gpu==1.12.0',
-        'autopool==0.1.0',
-        'wget==3.2',
-        'sox==1.3.7',
-        'sed_eval==0.2.1',
+        'numpy>=1.1',
+        'pandas>=0.25',
```
Comment (mrocamora) [inline setup.py:23]: Do we still need pandas?

File: .github/workflows/main.yml (hunk 1 above)
Comment (pzinemanas) [reply .github/workflows/main.yml:58]: Yes, we should. I'll add it.

File: setup.py (hunk 3 above)
Comment (pzinemanas) [reply setup.py:23]: No, we have to implement a `read_csv` function which is the only thing you are using from pandas.

**Pull request merged**